presslabs/wordpress-integration
# Update uploads via FTP

Uploads can now be served via FTP. 

Because PHP has native support for PHP stream wrappers, we add a filter to append `ftp://<host>:<port>` to each upload path.
We do that in order to support multiple uploads sources via rclone ftp server.

Fixes #10 

TODOs:
-  [x] test delete photo
-  [x] test delete pdf

#### Tests
* `git checkout uploads-via-ftp`
* in `.env` set `GCS_BUCKET`, `GCS_PROJECT_ID` and `GCS_SERVICE_ACCOUNT_JSON_KEY_PATH` to your local service account json key
* `docker-compose up`
* upload a photo (keep in mind that the docker-compose is using an old version of rclone)
* upload another photo
* you **should** see the second photo in the bucket, with the thumbnails generated
* delete the photo
* in the bucket, the photo and associated thumbnails **should** be deleted
